### PR TITLE
added install script for the impatient.

### PR DESCRIPTION
**Pull request by jollyjinx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,9 @@ This will kick out a shared library file `liberate-fileserver.dylib` that must b
...
 
-Now inject this library into the AppleFileServer daemon by adding the following lines to the `/System/Library/LaunchDaemons/com.apple.AppleFileServer.plist` just inside the first `<dict>` tag:
+Now inject this library into the AppleFileServer daemon by adding the liberate fileserver library to the dynamic loading path of apples fileserver:
 
-    <key>EnvironmentVariables</key>
-    <dict>    
-        <key>DYLD_INSERT_LIBRARIES</key>
-        <string>/usr/local/lib/liberate-fileserver.dylib</string>
-    </dict>
+    sudo defaults write /System/Library/LaunchDaemons/com.apple.AppleFileServer.plist  EnvironmentVariables -dict DYLD_INSERT_LIBRARIES /usr/local/lib/liberate-fileserver.dylib
```
Comment (joshado) [inline README.md:32]: Nice one, thanks. I'll merge this in now.

I'm still on 10.7 (can't bring myself to upgrade). Does it still work on 10.8?

Comment (jollyjinx) [inline README.md:32]: Yes, works fine on 10.8 and ZFS

**Comment by sbytnar** [comment]
Using "defaults write" has an undesirable side effect of converting the plist file from "XML  document text" (human readable) to "Apple binary property list" (binary blob).

Even with this change, liberate-fileserver.dylib does not present MacZFS 74.1.0 volumes on my Mountain Lion Server 10.8.2 installation.

What ZFS vendor/version is working for you?

**Comment by jollyjinx** [comment]
I've fixed the conversion back to xml.
I'm using ZEVO's ZFS implementation and it's working fine after specifying the ZFS root in Sharing Preferences on 10.8.2

**Pull request merged**